add-product/main.c

```diff
@@ -1,8 +1,25 @@
 #include <stdio.h>
+#include <stdlib.h>
+
+#include "../common/arg.h"
+#include "../common/util.h"
+
+static void
+usage(void)
+{
+	printf("%s [-h] [-p PRICE] [-q QTY] [-c COST] -t TYPE NAME\n", argv0);
+}
 
 int
-main(void)
+main(int argc, char **argv)
 {
-	printf("hello, world\n");
+	unsigned price, qty, cost;
+	const char *type;
+	ARGBEGIN {
+	case 'h':
+		usage();
+		exit(0);
+		break;
```

Comment:
Do we need a break after exit?

Reply:
normally not, but I maintain it in case of refactoring later

Reply:
I think there won't be any other case for -h. Would just do exit.